Modules/LinearBoltzmannSolver/SweepChunks/lbs_sweepchunk_pwl.h

```diff
@@ -113,7 +113,7 @@ class LBSSweepChunkPWL : public chi_mesh::sweep_management::SweepChunk
       int num_dofs = cell_fe_view->dofs; 
       auto transport_view =
         static_cast<LinearBoltzmann::CellViewFull*>((*grid_transport_view)[cell->local_id]);
-      auto sigma_tg = (*xsections)[transport_view->xs_id]->sigma_tg;
+      const auto & sigma_tg = (*xsections)[transport_view->xs_id]->sigma_tg;
```

Comment:
This needs to be updated according to the new development branch. `xsections` is now a reference to a `vector` of shared pointers. So the line should eventually look like this:
```
const auto& sigma_tg = xsections[transport_view.xs_id]->sigma_tg;
```

Reply:
should be up to date now